src/mumble/JackAudio.cpp

```diff
@@ -232,6 +235,7 @@ JackAudioSystem::JackAudioSystem() : bAvailable(false), users(0), client(nullptr
 	RESOLVE(jack_port_by_name)
 	RESOLVE(jack_port_flags)
 	RESOLVE(jack_port_get_buffer)
+	RESOLVE(jack_port_rename)
```

Comment:
I believe its been part of the API ever since the first release of JACK2. However, I never actually ended up using that export in this implementation, so it can be removed if compatibility is of express concern.

Reply:
Alright then. I'd be fine with leaving it in.